HaxmLoader/HaxmLoader.vcxproj

```diff
@@ -118,4 +118,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
There is no end of new line. Is it reasonable?

Reply:
Not desirable but I don't think it was me... I think I just let Visual Studio upgrade the project.

Reply:
I ever met this as well, no change, but it was marked.